src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Rules/AttributesMustMatchTests.cs

```diff
@@ -219,42 +219,6 @@ public class First {}
 new CompatDifference[] {
     CompatDifference.CreateWithDefaultMetadata(DiagnosticIds.CannotChangeAttribute, "", DifferenceType.Changed, "T:CompatTests.First:[T:CompatTests.FooAttribute]")
 }
```

Comment:
Since we desire this behavior can we add an explicit test for it?  That should avoid having this discussion again in the future :)

Reply:
To clarify, do you mean adding a test that asserts that changes to internal type arguments are caught?

Reply:
Yeah, that might catch if we had an inconsistent namespace between the internal type argument on left and right.

Reply:
Shouldn't we only validate internal members when this flag is set to true? https://github.com/dotnet/sdk/blob/412df6f0ccee8f0f309d88b674b9c8d6d88c58ea/src/ApiCompat/Microsoft.DotNet.ApiCompatibility/ApiComparerSettings.cs#L23-L26

Reply:
> The concept of internal/public on attributes is not well defined

Based on the previous behavior, right now, `IncludeInternalSymbols` applies to the attribute class as well. If we didn't want to do that, then I can remove that behavior as well.

Reply:
[We still filter out attributes which aren't visible outside of the assembly](https://github.com/dotnet/sdk/blob/65c6944d81aff5bdb7c9cf14637b8657a0c0a44d/src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/AttributesMustMatch.cs#L60-L63), even with this PR; but we don't filter out visible attributes which contain internal members anymore. That's the desired behavior, correct?